### PR TITLE
feat: add feature to zoom webview contents, closes #388

### LINKED ISSUE #388: Zoom (change scaling factor)
**Is your feature request related to a problem? Please describe.**
Currently, it is not possible to zoom the webview (e.g. via Ctrl +, Ctrl -, or Ctrl mousewheel)  as expected since every (to me) known browser supports it.

**Describe the solution you'd like**
1) Automatically enable the typical shortcuts for zooming in and out
2) typescript/javascript method for changing the scaling factor (zoom).

**Describe alternatives you've considered**
Currently, the only suggestion was to change the CSS property `document.style.zoom`.

### PR DESCRIPTION
**Pull request by keiya01**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [x] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [x] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: .changes/impl-zoom-feature.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"wry": minor
+---
+
+Add feature to zoom webview contents, closes #388.
```
Comment (amrbashir) [inline .changes/impl-zoom-feature.md]:
```suggestion
Add `WebView::zoom` method.
```

File: src/webview/mod.rs [hunk 2]
```diff
@@ -491,6 +491,10 @@ impl WebView {
     self.window.inner_size()
   }
 
+  pub fn zoom(&self, scale_factor: f64) {
```
Comment (amrbashir) [inline src/webview/mod.rs:501]:
```suggestion
  /// Set the webview zoom level
  ///
  /// ## Platform-specific:
  ///
  /// - **Android / iOS**: Not supported.
  pub fn zoom(&self, scale_factor: f64) {
```

**Review by amrbashir — CHANGES_REQUESTED** [review]
LGTM, but it is missing documentation and the change file needs a bit of rewording

**Review by amrbashir — DISMISSED** [review]

**Comment by keiya01** [comment]
Thank you for your review. I fixed your review.

**Comment by keiya01** [comment]
Sorry I dismissed your approve by my commit :(

**Review by amrbashir — DISMISSED** [review]
No worries

**Comment by amrbashir** [comment]
Just noticed this, but you should probably [sign your commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

**Comment by keiya01** [comment]
Sure, I pushed signed commit.

**Review by amrbashir — APPROVED** [review]

File: src/webview/wkwebview/mod.rs [hunk 3]
```diff
@@ -584,6 +584,13 @@ r#"Object.defineProperty(window, 'ipc', {
       (view_frame.size.width as f64, view_frame.size.height as f64).into();
     logical.to_physical(scale_factor)
   }
+
+  pub fn zoom(&self, scale_factor: f64) {
+    #[cfg(target_os = "macos")]
+    unsafe {
+      let _: () = msg_send![self.webview, setPageZoom: scale_factor];
```
Comment (amrbashir) [inline src/webview/wkwebview/mod.rs:590]: According to https://developer.apple.com/documentation/webkit/wkwebview/3516411-pagezoom?language=objc , this api seems to be available on iOS 14+. And is only available in macOS 11+, should we document that @wusyong?

Comment (wusyong) [reply src/webview/wkwebview/mod.rs:590]: Yes, please add it platform specific doc.

File: src/webview/wkwebview/mod.rs [hunk 4]
```diff
@@ -584,6 +584,13 @@ r#"Object.defineProperty(window, 'ipc', {
       (view_frame.size.width as f64, view_frame.size.height as f64).into();
     logical.to_physical(scale_factor)
   }
+
+  pub fn zoom(&self, scale_factor: f64) {
+    #[cfg(target_os = "macos")]
```
Comment (amrbashir) [inline src/webview/wkwebview/mod.rs]:
```suggestion
```

File: src/webview/mod.rs [hunk 5]
```diff
@@ -491,6 +491,15 @@ impl WebView {
     self.window.inner_size()
   }
 
+  /// Set the webview zoom level
+  ///
+  /// ## Platform-specific:
+  ///
+  /// - **Android / iOS**: Not supported.
```
Comment (amrbashir) [inline src/webview/mod.rs]:
```suggestion
  /// - **Android**: Not supported.
  /// - **macOS**: available on macOS 11+ only.
  /// - **iOS**: available on iOS 14+ only.
```

**Review by amrbashir — CHANGES_REQUESTED** [review]

**Review by amrbashir — APPROVED** [review]

**Review by wusyong — APPROVED** [review]

**Pull request merged**